TwilioVerify.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'TwilioVerify'
-  s.version = '0.4.0'
+  s.version = '1.0.0'
```

Comment:
This is not needed, CD will update it when you merge this PR using a commit message like this:
title: `feat: xcframework support`
body: `BREAKING CHANGE: TwilioVerify renamed to TwilioVerifySDK  to prevent bug https://bugs.swift.org/browse/SR-14195`

Reply:
@yafuquen Thank you, I will revert that change. 👍